desktop/cmp/treemap/TreeMap.js

```diff
@@ -0,0 +1,297 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import PT from 'prop-types';
+import {Highcharts, highchartsExporting, highchartsOfflineExporting, highchartsExportData, highchartsTree, highchartsHeatmap} from '@xh/hoist/kit/highcharts';
+
+import {XH, elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
+import {div, box} from '@xh/hoist/cmp/layout';
+import {Ref} from '@xh/hoist/utils/react';
+import {resizeSensor} from '@xh/hoist/kit/blueprint';
```

Comment:
Is this the dep that's keeping this in the desktop package?  Anything else that we think is a real blocker to usage on mobile as well?  

Even if we don't test / recommend up front, it would be a nice win if we could land this in the cross platform `/cmp/` package to set us up for full mobile support now or later. We've seen how nice these maps can look even on phone.